docs/reference-architectures/app-service-web-app/baseline-zone-redundant-content.md

```diff
@@ -224,28 +224,28 @@ The App Service baseline configures authentication and authorization for user id
 
 :::image type="complex" source="images/baseline-app-service-deployments.svg" lightbox="images/baseline-app-service-deployments.svg" alt-text="Diagram that shows a baseline App Service deployment architecture.":::
-    The diagram shows a subnet containing self-hosted deployment agents. It also adds an Azure Pipelines with managed agents. The last change is numbered for the deployment flow. Number one is on Azure Pipelines. Number two is an arrow from the self-hosted agents to Azure Pipelines. Three is an arrow from the self-hosted agent to the private endpoint for Azure Storage. Four is again above Azure Pipelines and the managed agents. Five is in App Services. Six is again over Azure Pipelines and the managed agents.
+    The diagram shows a subnet containing self-hosted deployment agents. It also adds Azure Pipelines with managed agents. The last change is numbered for the deployment flow. Number one is on Azure Pipelines. Number two is an arrow from the self-hosted agents to Azure Pipelines. Three is an arrow from the self-hosted agent to the private endpoint for Azure Storage. Four is again above Azure Pipelines and the managed agents. Five is in App Services. Six is again over Azure Pipelines and the managed agents.
 :::image-end:::
 *Figure 3: Deploying Azure App Service applications*
 
 ### Deployment flow
 
 1. As part of the release pipeline, the pipeline posts a job request for the self-hosted agents in the job queue. The job request is for the agent to upload the *publish zip file* build artifact to an Azure Storage Account.
 2. The self-hosted deployment agent picks up the new job request through polling. It downloads the job and the build artifact.
-3. The self-hosted deployment agent uploads the zip file to the storage account through the storage account private endpoint.
+3. The self-hosted deployment agent uploads the zip file to the storage account through the storage account's private endpoint.
 4. The pipeline continues, and a managed agent picks up a subsequent job. The managed agent [makes a CLI call to update the appSetting](/cli/azure/webapp/config/appsettings) WEBSITE_RUN_FROM_PACKAGE to the name of the new publish zip file for the staging slot.
 
     ```bash
     az webapp config appsettings set -g MyResourceGroupName -n MyUniqueApp --slot staging --settings WEBSITE_RUN_FROM_PACKAGE=UriToNewZip
     ```
 
 5. Azure App Service pulls the new publish zip file from storage via the storage account private endpoint. The staging instance restarts with the new package because WEBSITE_RUN_FROM_PACKAGE was set to a different file name.
-6. The pipeline resumes and runs any smoke tests or waits for approval. If the tests pass or the approval is given, the pipeline swaps the staging and production slots.
+6. The pipeline resumes and runs any smoke tests or waits for approval. If the tests pass or approval is given, the pipeline swaps the staging and production slots.
 
-### Deployment guidance
+### Deployment Guidance
```

Comment:
Our style guide requires headings to be sentence cased: https://review.learn.microsoft.com/en-us/help/contribute/contribute-how-to-write-pull-request-quality-criteria?branch=main#nonblocking

Reply:
Gotcha - thanks - Updated.